proto/api/workload/workload.proto

```diff
@@ -1,4 +1,5 @@
 syntax = "proto3";
+package spire.api.workload;
```

Comment:
You mentioned that you ran into a compatibility issue when the package name was mismatched...? I realize now that if package names must match for compatibility, then we probably need to specify the name in the SPIFFE workload api spec... and if that's the case, this name should probably be `spiffe.api.workload`.